SerialPrograms/Source/PokemonLA/Inference/PokemonLA_ShinySoundDetector.cpp

```diff
@@ -0,0 +1,78 @@
+/*  Shiny Sound Detector
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#include "CommonFramework/Tools/AudioFeed.h"
+#include "PokemonLA_ShinySoundDetector.h"
+
+#include <QString>
+
+#include <cfloat>
+#include <iostream>
+using std::cout;
+using std::endl;
+
+namespace PokemonAutomation{
+namespace NintendoSwitch{
+namespace PokemonLA{
+
+
+
+ShinySoundDetector::ShinySoundDetector(Logger& logger)
+    : AudioInferenceCallback("ShinySoundDetector")
+    , m_logger(logger)
+{
+    // TODO : handle case with sample rate mismatch between file and capture card:
+
+    // TODO: move the template file to the resource folder
+    QString shinyFilename = "./heracrossShinyTemplateCompact.wav";
```

Comment:
Let's go ahead and move this to the resource folder. Since we can't find an official asset from the data dump, let's just use this then.